ambry-tools/src/main/java/com.github.ambry/store/IndexProcessingHelper.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2018 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+import com.github.ambry.config.StoreConfig;
+import com.github.ambry.utils.Pair;
+import com.github.ambry.utils.Throttler;
+import com.github.ambry.utils.Time;
+import java.io.File;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Helper class used for processing indexes and batching conversion maps of store keys
+ */
+class IndexProcessingHelper {
```

Comment:
minor: this can probably be public since it seems like a general tool helper

Reply:
Has to be public if you decide to send an instance of this into `ConsistencyCheckerTool` (constructors and package private functions have to be made public too)

Reply:
See suggestions to drop IndexProcessingHelper altogether